SpringProject/src/it/java/com/gradproject2019/user/UserControllerIT.java

```diff
@@ -38,13 +37,18 @@
     private String baseUri;
     private URI uri;
     private URI uri2;
+    private URI uri3;
+    private URI uri4;
 
     @Before
     public void setUp() throws URISyntaxException {
         universalSetUp();
         baseUri = "http://localhost:" + testServerPort + "/users";
         uri = new URI(baseUri);
         uri2 = new URI(baseUri + "/" + savedUser.getId());
+        uri3 = new URI(baseUri  +"?manager_id=" + savedManager.getId());
```

Comment:
I'm pretty sure you can dynmically build queries using the spring testing framework

Reply:
I'll make a new ticket to update our testing as we've got some other tests to update as well!